bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/impl/GemsStrategy.scala

```diff
@@ -105,6 +105,9 @@ trait GemsStrategy extends AgsStrategy {
     cans ++ odgw
   }
 
+  // TODO Implement for GEMS
+  override protected [ags] def analyze(ctx: ObsContext, mt: MagnitudeTable, guideProbe: ValidatableGuideProbe, guideStar: SiderealTarget): Option[AgsAnalysis] = None
```

Comment:
This may be the reason why this wasn't implemented before.  Will it be possible to validate one probe in isolation when they depend upon each other as for Canopus?